app/views/users/invitations/_new.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="modal fade" id="newUser">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="newUserlTitle">Send Invitation</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <%= form_for(resource, as: resource_name, url: invitation_path(resource_name),remote: true, 
+                    html: { method: :post }) do |f| %>
+          <div id='user_invitation'></div>
+
+          <div class="form">
+            <div class="notice" >
+              <%= @notice %>
+            </div>
+            
+            <div class="error" ><%= resource.errors[:first_name][0] %></div>
```

Comment:
closing div should be below the opening div, as well as properly indent it like what you have done above from line number 16 to line no 18